easybuild/easyconfigs/g/gdbm/gdbm-1.18.1-foss-2020a.eb

```diff
@@ -0,0 +1,28 @@
+easyblock = 'ConfigureMake'
+
+name = 'gdbm'
+version = '1.18.1'
+
+homepage = 'https://www.gnu.org.ua/software/gdbm/'
+description = """GNU dbm (or GDBM, for short) is a library of 
+database functions that use extensible hashing and work similar to the standard UNIX dbm. 
+These routines are provided to a programmer needing to create and manipulate a hashed database."""
+
+toolchain = {'name': 'foss', 'version': '2020a'}
+
+source_urls = ['https://ftp.gnu.org/gnu/gdbm']
+sources = ['%(name)s-%(version)s.tar.gz']
+checksums = ['86e613527e5dba544e73208f42b78b7c022d4fa5a6d5498bf18c8d6f745b91dc']
+
+dependencies = []
+
+sanity_check_commands = ['gdbmtool --help']
```

Comment:
Style nitpicking: `sanity_check_commands` usually goes *below* `sanity_check_paths`, with an empty line in between

Reply:
I saw that in other configs, my sense of aesthetics argued otherwise :D Changing it back